vm/src/stdlib/math.rs

```diff
@@ -510,6 +510,29 @@ fn math_factorial(value: PyIntRef, vm: &VirtualMachine) -> PyResult<BigInt> {
     Ok(product)
 }
 
+fn math_perm(n: PyIntRef, k: OptionalArg<PyIntRef>, vm: &VirtualMachine) -> PyResult<BigInt> {
```

Comment:
```suggestion
fn math_perm(n: PyIntRef, k: OptionalArg<Option<PyIntRef>>, vm: &VirtualMachine) -> PyResult<BigInt> {
```
`Option<PyIntRef>` means `None` or `int`

Reply:
Thanks!!